tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -113,5 +128,37 @@ nest_plz = (
     </div>
   </div>
 );
+
+regression_not_transformed_1 = (
+  <span>
+    {\" \"}<Icon icon=\"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\" />
+  </span>
+);
+
+regression_not_transformed_2 = (
+  <span>
+    <Icon icon=\"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\" />{\" \"}
+  </span>
+);
+
+similar_1 = (
```

Comment:
similar_1, similar_2, and similar_3 aren't necessary to verify the current implementation, but might be handy later. thoughts on over-testing?